<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profile</class>
 <widget class="QDialog" name="profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Profile</string>
  </property>
  <widget class="QLabel" name="profileTitle">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>User Profile</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>40</y>
     <width>361</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="username">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>First Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="firstName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Last Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lastName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="email"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Total Spent:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="totalSpent">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="updatebutton">
        <property name="text">
         <string>Update Profile</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>updatebutton</sender>
   <signal>clicked()</signal>
   <receiver>profile</receiver>
   <slot>updatedProfile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updatedProfile()</slot>
 </slots>
</ui>
